api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -214,6 +214,38 @@ message OktaAssignmentMetadata {
   string EndingStatus = 4 [(gogoproto.jsontag) = "ending_status,omitempty"];
 }
 
+// AccessListMemberMetadata contains common metadata for access list member resource events.
+message AccessListMemberMetadata {
+  // joined_on is the date that the member joined.
+  google.protobuf.Timestamp joined_on = 1 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "session_start,omitempty"
+  ];
+
+  // removed_on is the date that the access list member was removed.
+  google.protobuf.Timestamp removed_on = 2 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "session_start,omitempty"
+  ];
+
+  // reason is the reason that the member was added, modified, or removed.
+  string reason = 3;
+
+  // access_list_name is the name of the access list the member is being added to or removed from.
+  string access_list_name = 4 [(gogoproto.jsontag) = "access_list_name,omitempty"];
+
+  // member_name is the name of the member.
+  string member_name = 5 [(gogoproto.jsontag) = "member_name,omitempty"];
+}
+
+// AccessListReviewMetadata contains metadata for access list review events.
+message AccessListReviewMetadata {
+  // message is the message that was supplied during the review.
+  string message = 1;
```

Comment:
No JSON tags?

Reply:
Truth be told I'd rather leave these using the defaults as much as possible as opposed to using `gogo` specific stuff. The defaults for these in particular should be pretty safe here.